src/components/Blog/Blog.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+import './Blog.css'
+
+export function Blog() {
+  return (
+    <article className="blog">
+      <h2>blog</h2>
+      <div className="blog__content">
+        <p>
+          Find out about open source with our <Link to="/blog">blog</Link>.
+        </p>
+      </div>
+    </article>
```

Comment:
I think, we should set a specific license for the white paper (for example Creative Commons)